test/e2e/specs/linksConsoleErrorsEsmPage.spec.ts

```diff
@@ -50,3 +57,16 @@ function handleCommonEsmBrowsersErrors(linkName: string, consoleErrors: ConsoleM
             validatePageErrors(consoleErrors, [], ['the server responded with a status of 404']);
     }
 }
+
+/**
+ * Waits for a deploy preview URL to become available by making repeated requests and check that link is visible.
+ */
+async function waitForDeployPreviewUrl(link: ExampleLinkType, page: Page): Promise<void> {
+    console.log('Waiting for deploy preview to be ready...');
```

Comment:
```suggestion

```

Reply:
The print was added to get indication that we're waiting for the preview and not just wait long time without any indication which can feel like something is stuck.

Reply:
Print was removed